lib/server.js

```diff
@@ -185,6 +185,10 @@ internals.Server.prototype.initialize = function (callback) {
         return errorCallback(new Error('Cannot start server before plugins finished registration'));
     }
 
+    if (this._state === 'initialized') {
+        return callback();
```

Comment:
Technically it should use the `errorCallback` after renaming it to `nextTickCallback`